frontend/projects/valtimo/components/src/lib/components/carbon-list/carbon-list.component.ts

```
@@ -477,67 +485,125 @@ export class CarbonListComponent implements OnInit, AfterViewInit, OnDestroy {
+    const row = [
+      ...this.getDragAndDropItemsItems(item, index, length),
+      ...fields.map((field: ColumnConfig) => {
+        switch (field.viewType) {
+          case ViewType.TEMPLATE:
+            return new TableItem({
+              data: {item, index, length, ...field.templateData},
+              item,
+              template: field.template,
+            });
+          case ViewType.BOOLEAN:
+            let data = this.resolveObject(field, item);
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)